src/web/crate_details.rs

```diff
@@ -420,6 +424,167 @@ pub(crate) async fn get_all_releases(
     Ok(res.into_response())
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize)]
+struct ShortMetadata {
+    name: String,
+    version_or_latest: String,
+    doc_targets: Vec<String>,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize)]
+struct PlatformList {
+    metadata: ShortMetadata,
+    inner_path: String,
+    use_direct_platform_links: bool,
+    current_target: String,
+}
+
+impl_axum_webpage! {
+    PlatformList = "rustdoc/platforms.html",
+    cpu_intensive_rendering = true,
+}
+
+#[tracing::instrument]
+pub(crate) async fn get_all_platforms(
+    Path(params): Path<RustdocHtmlParams>,
+    Extension(pool): Extension<Pool>,
+    uri: Uri,
+) -> AxumResult<AxumResponse> {
+    // since we directly use the Uri-path and not the extracted params from the router,
+    // we have to percent-decode the string here.
+    let original_path = percent_encoding::percent_decode(uri.path().as_bytes())
+        .decode_utf8()
+        .map_err(|_| AxumNope::BadRequest)?;
+    let mut req_path: Vec<&str> = original_path.split('/').collect();
+
+    let release_found = match_version_axum(&pool, &params.name, Some(&params.version)).await?;
+    trace!(?release_found, "found release");
+
+    // Remove the empty start, "releases", the name and the version from the path
+    req_path.drain(..4).for_each(drop);
```

Comment:
Isn't this `params.path`?

Reply:
It is. Most of this code comes from `rustdoc_html_server_handler` so I tried to keep the same format.

Reply:
I updated this to use `params.path`.